.github/workflows/build-ios-release-pullrequest.yml

```
@@ -86,50 +93,67 @@ jobs:
         run: |
           if [ -d "~/Library/MobileDevice/Provisioning Profiles" ]; then
             echo "Provisioning profiles are available in the cache."
-            ls -la ~/Library/MobileDevice/Provisioning Profiles
+            ls -la ~/Library/MobileDevice/Provisioning\ Profiles
           else
             echo "Provisioning profiles directory does not exist."
           fi
+
       - name: Generate Build Number based on timestamp
         id: generate_build_number
         run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:46: Double quote to prevent globbing and word splitting

(shellcheck)